### PR TITLE
solved solution 1. ideal weight

### PR DESCRIPTION
**Pull request by neb5562**

### CODE REVIEW

**Comment by Giga-Chubinidze** [comment]
Great Job!

File: 1 - Ideal Weight/solution_1.rb [hunk 1]
```diff
@@ -1,3 +1,30 @@
 # Ideal weight. The program asks for name and height and displays the ideal weight using the formula 
 # <height> - 110, then displays the result to the user by name. If the ideal weight is negative, the line 
 # "Your weight is already optimal" is displayed
+
+class Person
+    def initialize(name, height)
```
Comment (alexshvedko) [inline 1 - Ideal Weight/solution_1.rb:6]: https://clearwater.readthedocs.io/en/stable/Clearwater_Ruby_Coding_Guidelines.html
Use 2 space indent, no tabs.

File: 1 - Ideal Weight/solution_1.rb [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
 # "Your weight is already optimal" is displayed
+
+class Person
+  def initialize(name, height)
+    @name = name
+    @height = height
+  end
+
+  def ideal_weight
+    @i_weight = @height.to_i - 110
```
Comment (alexshvedko) [inline 1 - Ideal Weight/solution_1.rb:12]: magic number

File: 1 - Ideal Weight/solution_1.rb [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
 class Person
-  FIXED_VALUE_TO_SUBSCRABT = 110
+  FIXED_VALUE_TO_SUBSCRAPT = 110
   def initialize(name, height)
     @name = name
     @height = height
   end
 
   def ideal_weight
-    @i_weight = @height.to_i - FIXED_VALUE_TO_SUBSCRABT
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb]: Same here

File: 1 - Ideal Weight/solution_1.rb [hunk 4]
```diff
@@ -9,11 +9,10 @@ def initialize(name, height)
     @height = height
   end
 
-  def ideal_weight
-    @i_weight = @height.to_i - FIXED_VALUE_TO_SUBSCRAPT
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb]: changed `@i_weight to @ideal_weight`

File: 1 - Ideal Weight/solution_1.rb [hunk 5]
```diff
@@ -9,11 +9,10 @@ def initialize(name, height)
     @height = height
   end
 
-  def ideal_weight
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb]: changed  `def ideal_weight ` method name to  ` "def get_ideal_weight`

File: 1 - Ideal Weight/solution_1.rb [hunk 6]
```diff
@@ -9,11 +9,10 @@ def initialize(name, height)
...
-    @i_weight = @height.to_i - FIXED_VALUE_TO_SUBSCRAPT
-    return 'Your weight is already optimal' if @i_weight.negative?
-
-    "Name: #{@name}\nHeight: #{@height}\nIdeal weight: #{@i_weight}"
+  def get_ideal_weight
+    @ideal_weight = @height.to_i - FIXED_VALUE_TO_SUBSCRAPT
+    return 'Your weight is already optimal' if @ideal_weight.negative?
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:16]: `@i_weight` to `@ideal_weight`

File: 1 - Ideal Weight/solution_1.rb [hunk 7]
```diff
@@ -27,4 +26,4 @@ def ideal_weight
 person = Person.new name, height
 
 # print ideal weight
-puts person.ideal_weight
+puts person.get_ideal_weight
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:33]: changed `ideal_weight` method name to `get_ideal_weight`

File: 1 - Ideal Weight/solution_1.rb [hunk 8]
```diff
@@ -4,6 +4,7 @@
 
 class Person
   FIXED_VALUE_TO_SUBSCRAPT = 110
+
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:7]: added empty line for better view

File: 1 - Ideal Weight/solution_1.rb [hunk 9]
```diff
@@ -20,6 +21,7 @@ def get_ideal_weight
 # get name from input
 puts 'Please enter your name:'
 name = gets.chomp
+
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:24]: same here

File: 1 - Ideal Weight/solution_1.rb [hunk 10]
```diff
@@ -8,17 +8,19 @@ def initialize(name, height)
     @name = name
     @height = height
   end
-
+  
+  # calculate and return ideal weight
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:13]: added method description comment

File: 1 - Ideal Weight/solution_1.rb [hunk 11]
```diff
@@ -3,13 +3,14 @@
 # "Your weight is already optimal" is displayed
 
 class Person
+  FIXED_VALUE_TO_SUBSCRAPT = 110
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb:6]: Added constant instead of magic number

File: 1 - Ideal Weight/solution_1.rb [hunk 12]
```diff
@@ -13,21 +9,17 @@ def initialize(name, height)
   # calculate and return ideal weight
   def get_ideal_weight
     @ideal_weight = @height.to_i - FIXED_VALUE_TO_SUBSCRAPT
-    return 'Your weight is already optimal' if @ideal_weight.negative?
```
Comment (neb5562) [inline 1 - Ideal Weight/solution_1.rb]: removed `return`

**Pull request closed without merging**